base_multi_company/models/company_active_mixin.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class CompanyActiveMixin(models.AbstractModel):
+
+    _name = 'company.active.mixin'
+    _description = 'Company Active Mixin'
+
+    active = fields.Boolean(
+        compute='_compute_company_active_status',
+        inverse='_inverse_company_active_status',
```

Comment:
Note these are out of compliance with OCA guidelines for method naming in order to avoid potential name clashes during inheritance. I'm open to other suggestions for naming though.